Curs/src/views/index.js

```diff
@@ -1,19 +1,12 @@
 
-interviewApp = {}
+interviewApp = {}; // interviewApp object
```

Comment:
Without the comment, I'd have no idea what this is, judging solely by the code... :stuck_out_tongue: (I'm obviously ironic here!)

Reply:
Yeah but you know..

I have no idea who will see this code and.. I want to be sure..I'm sure you understand me.

Also, is a good practice to have the code commented.

Reply:
:+1: